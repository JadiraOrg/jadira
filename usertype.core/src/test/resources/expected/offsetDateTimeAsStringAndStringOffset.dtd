<!ELEMENT dataset (
    OFFSETDATETIMEASSTRINGANDSTRINGOFFSET*)>

<!ELEMENT OFFSETDATETIMEASSTRINGANDSTRINGOFFSET EMPTY>
<!ATTLIST OFFSETDATETIMEASSTRINGANDSTRINGOFFSET
    ID CDATA #REQUIRED
    NAME CDATA #IMPLIED
    MY_DATE CDATA #IMPLIED
    MY_OFFSET CDATA #IMPLIED
>

