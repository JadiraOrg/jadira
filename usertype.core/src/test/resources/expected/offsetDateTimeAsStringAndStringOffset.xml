<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <offsetDateTimeAsStringAndStringOffset ID="0" NAME="test_0" MY_DATE="2004-02-25T17:03:45.00000076" MY_OFFSET="Z"/>
  <offsetDateTimeAsStringAndStringOffset ID="1" NAME="test_1" MY_DATE="1980-03-11T02:03:45.0" MY_OFFSET="+02:00"/>
</dataset>
